src/collections/_documentation/workflow/user-settings.md

```diff
@@ -0,0 +1,187 @@
+
+## Deploy Notifications
+
+Choose a Deploy Notifications option to keep track of deployment details like release version, environments, and commit reviews. You can fine-tune options from the dropdown next to each organization.
+
+{% asset user-settings/us-deploy.png alt="Deploy Notifications selection in Notifications under User Settings" width="673" %}
+
+## My Activity
+
+Updates regarding your user account actions, such as claiming unassigned issues, can be turned into an email notification in this section.
+
+{% asset user-settings/us-myact.png alt="My Activity selection in Notifications under User Settings" width="450" %}
+
+{% capture __alert_content -%}
+You can find a shortcut to the Emails panel at the bottom of this section.
+
+{% asset user-settings/us-npanellink.png alt="A link to the Email panel in Notifications under My Activity" width="465" %}
+
+{%- endcapture -%}
+{%- include components/alert.html
+    title="Note"
+    content=__alert_content
+%}
+
+## Emails
+
+The email address used to login to your Sentry account is by default your primary email address. You can add an alternative email address under the Add Secondary Emails section.
+
+{% asset user-settings/us-eprime.png alt="Primary email field in Emails under User Settings" width="494" %}
+
+{% capture __alert_content -%}
+
+You can find a shortcut to the Notifications panel at the bottom of this section.
+
+{% asset user-settings/us-epanellink.png alt="A link to the Notifications panel in Emails under Additional Email" width="450" %} 
+
+{%- endcapture -%}
+{%- include components/alert.html
+    title="Note"
+    content=__alert_content
+%}
+
+## Subscriptions
+
+As part of our compliance with the EU’s General Data Protection Regulation (GDPR), you wills only receive marketing campaign emails after explicitly opting-in to any of the listed categories. 
+
+{% asset user-settings/us-subs.png alt="Subscriptions section under User Settings" width="450" %}
+
+## Authorized Applications
+
+Sentry lists all authorized third-party applications in this section and requires authentication tokens for authorization. You can find tokens in the [*Auth Tokens*](https://sentry.io/api/) section. 
+
+## Identities
+
+Sentry integrates with several identity providers, like Okta, OneLogin, Auth0, and Yubico. To disconnect any previously configured provider, click the "Disconnect" button.
+
+{% asset user-settings/us-id.png alt="Disconnect button in Identities section under User Settings" %}
+
+## Close Account
+
+Closing your Sentry account automatically removes all data associated with your account. If your account is the sole owner of an organization, this organization will be deleted. Organizations with multiple owners will remain unchanged.
+
+{% asset user-settings/us-close.png alt="Close Account button in Close Account section under User Settings" width="745" %}
+
+{% capture __alert_content -%}
+
+Deleting your account cannot be undone. 
```

Comment:
Should this be bolded or highlighted in some manner?

Reply:
Yep! It's inside of an alert box. It will look like this in production:

![image](https://user-images.githubusercontent.com/25088225/80145805-9b2f9780-8565-11ea-9aa7-a166e92aa35b.png)